### PR TITLE
Added file aliquot_sum.py

### PR DESCRIPTION
**Pull request by ken437**
### **Describe your change:**
I added a new algorithm in the maths directory that finds the aliquot sum of a number.
Algorithm explanation: https://en.wikipedia.org/wiki/Aliquot_sum

* [X] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [X] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [X] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [X] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [N/A] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: maths/aliquot_sum.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from typing import Union
+
+
+def aliquot_sum(input_num: int) -> Union[int, str]:
```
Comment (cclauss) [inline maths/aliquot_sum.py:4]: Let’s always return an int but let’s raise a [ValueError](https://github.com/TheAlgorithms/Python/blob/Master/maths/factorial_python.py#L22] on nonpositive ints.

File: maths/aliquot_sum.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    >>> aliquot_sum(15)
+    9
+    >>> aliquot_sum(6)
+    6
+    >>> aliquot_sum(-1)
+    'Please enter a positive integer'
+    >>> aliquot_sum(12)
+    16
+    >>> aliquot_sum(19)
+    1
```
Comment (cclauss) [inline maths/aliquot_sum.py:25]: Please add `aliquot_sum(0)` and `aliquot_sum(1.7)`.

File: maths/aliquot_sum.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    """
+    if input_num <= 0:
+        return "Please enter a positive integer"
+    sum = 0
+    for divisor in range(1, input_num):
+        if input_num % divisor == 0:
+            sum += divisor
+    return sum
```
Comment (cclauss) [inline maths/aliquot_sum.py:33]:
```suggestion
    return sum(divisor for divisor in range(1, input_num) if input_num % divisor == 0)
```

File: maths/aliquot_sum.py (hunk 2 above)
Comment (ken437) [reply maths/aliquot_sum.py:25]: Is there a way to directly modify the pull request without making a new one? I'm kind of new to Git.

Comment (ken437) [reply maths/aliquot_sum.py:29]: Never mind, I think I found out how

**Comment by ken437** [comment]
> Well done! Nice contribution. Thanks for doing this.

You're welcome!

**Pull request merged**